code/modules/mob/living/carbon/xenomorph/Abilities.dm

```diff
@@ -29,6 +29,10 @@
 		to_chat(X, SPAN_XENOWARNING("You can't do that from there."))
 		return
 
+	if(SSticker?.mode?.hardcore)
```

Comment:
hardcore should be set on the xenos themselves too, more straightforward

Reply:
Pretty sure setting hardcore checks on xenos for making tunnels is less straightforward with minimal differences and with more code while giving a hardcore check only on the ability takes the least amount of code.

if anything, hardcore requires to be refactored entirely but this is out of scope of the PR